src/plugins/console/public/styles/_app.scss

```diff
@@ -142,6 +146,8 @@
  */
 .console__monaco_editor__selectedRequests {
   background: transparentize($euiColorPrimary, .9);
+  border: solid;
```

Comment:
You might be able to get by with...
```
border: $euiBorderThin
```
which sets a solid 1px border and then remove the next line.

Reply:
Sure, I will add `border: $euiBorderThin`! But if we remove the next line, the color would not be `primary` (it would be grayish)?